lib/chef_zero/server.rb

```diff
@@ -111,7 +111,7 @@ module ChefZero
   class Server
 
     DEFAULT_OPTIONS = {
-      :host => '127.0.0.1',
+      :host => ['127.0.0.1'],
```

Comment:
in this day and age should `::1` get dropped in here by default or are we not there yet?